README.md

```diff
@@ -21,34 +21,35 @@ Service also written in Connect Go.
 To verify compatibility with other protocol implementations, the conformance test also uses reference client
 and server implementations that use the [gRPC-Go module](https://github.com/grpc/grpc-go).
 
-Also, please not that this project is currently in alpha. As a result, the conformance suite makes no backward compatibility
-guarantees at this point. The goal is to eventually publish a stable release but please be aware we may make changes
+Also, please note that this project is currently in alpha. As a result, it does not currently make backward compatibility
+guarantees. The goal is to eventually publish a stable release but please be aware we may make changes
```

Comment:
We may want to call it something other than alpha since the first version we create will be "v1.0.0-rc1". Maybe state that it is "pre-v1.0.0" instead?